source/extensions/filters/http/local_ratelimit/local_ratelimit.h

```diff
@@ -57,6 +64,13 @@ class FilterConfig : public ::Envoy::Router::RouteSpecificFilterConfig {
   const Router::HeaderParser& responseHeadersParser() const { return *response_headers_parser_; }
   Http::Code status() const { return status_; }
 
+  const std::vector<Envoy::RateLimit::Descriptor>& descriptors() const { return descriptors_; }
+  uint64_t stage() const { return stage_; }
+  envoy::extensions::filters::http::local_ratelimit::v3::LocalRateLimit_VhRateLimitsOptions
+  virtualHostRateLimits() const {
```

Comment:
if you really want this helper, return a const ref instead of a copy

Reply:
It's an enum.. so not passing in as const ref..